### PR TITLE
Feature/223 white label

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
 <head>
   <meta charset="UTF-8" />
-  <link rel="icon" type="image/svg+xml" href="/tonomy-logo48.png" />
+  <link rel="icon" type="image/svg+xml" href="/favicon.png" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Tonomy</title>
+  <title>Pangea</title>
```
Comment (theblockstalk) [inline index.html:8]: should come from settings file

**Review by theblockstalk — COMMENTED** [review]
You have statically white labelled everything. This needs to be dynamic, based on deployment.

Variables such as "Tonomy Staging" or "Pangea" should be in a config.json file
Images for all the environments should be put in a directory and referenced from the appropriate config.json file.

Another approach would be to have a directory with config.js file and all the images, for each environment. As done here https://github.com/telosnetwork/open-block-explorer/tree/master/src/config/chains

File: src/accounts/molecules/ShowQr.tsx [hunk 2]
```diff
@@ -12,14 +13,14 @@ export default function QROrLoading({
     <>
       {!isMobile() && (
         <>
-          <TP>Scan the QR code with the Tonomy ID app</TP>
+          <TP>Scan the QR code with the {settings.config.ecosystemName} app</TP>
```
Comment (theblockstalk) [inline src/accounts/molecules/ShowQr.tsx:16]: should be the App Name, not ecosystem name
Tonomy ID | Pangea Passport Testnet--> appName
Tonomy | Tonomy Staging | Pangea --> ecosystemName

Please check all locations.

File: src/accounts/pages/DownloadApp.tsx [hunk 3]
```diff
@@ -9,18 +9,21 @@ import playStoreBadge from "../assets/google-play-badge.png";
...
-      <TH4>The easiest and safest way to access Tonomy apps</TH4>
+      <TH3>{settings.config.ecosystemName}</TH3>
+      <TH4>
+        The easiest and safest way to access {settings.config.ecosystemName}{" "}
+        apps
+      </TH4>
       <TP className="paragraph">
-        Tonomy ID is a self-sovereign identity digital wallet that protects your
-        security and privacy. &nbsp;
+        {settings.config.ecosystemName} is a self-sovereign identity digital
```
Comment (theblockstalk) [inline src/accounts/pages/DownloadApp.tsx:18]: appName

File: src/common/config/config.json
Comment (theblockstalk) [inline src/common/config/config.json:1]: this should stay as the Tonomy ID Development app. For development environment

File: src/common/config/config.production.json [hunk 4]
```diff
@@ -1,17 +1,13 @@
...
-    "theme": {
-        "primaryColor": "#6200ee",
-        "secondaryColor": "#03dac4",
-        "tertiaryColor": "white",
-        "linkColor": "#0057B2"
-    },
-    "appLogoUrl": "/tonomy-logo48.png",
+    "themeFile": "theme.pangea.css",
+    "appLogoUrl": "/favicon.png",
     "appName": "Pangea",
```
Comment (theblockstalk) [inline src/common/config/config.production.json:4]: appName: UNITED Wallet

File: src/common/config/config.staging.json [hunk 5]
```diff
@@ -1,17 +1,13 @@
...
-        "primaryColor": "#6200ee",
-        "secondaryColor": "#03dac4",
-        "tertiaryColor": "white",
-        "linkColor": "#0057B2"
-    },
+    "themeFile": "theme.tonomy.css",
     "appLogoUrl": "/tonomy-logo48.png",
     "appName": "Tonomy ID Staging",
-    "ecosystemName": "Tonomy Staging",
+    "ecosystemName": "Tonomy ID",
```
Comment (theblockstalk) [inline src/common/config/config.staging.json:5]: ecosystemName: Tonomy Staging

File: src/common/config/config.testnet.json [hunk 6]
```diff
@@ -1,17 +1,13 @@
...
-        "secondaryColor": "#03dac4",
-        "tertiaryColor": "white",
-        "linkColor": "#0057B2"
-    },
-    "appLogoUrl": "/tonomy-logo48.png",
+    "themeFile": "theme.pangea.css",
+    "appLogoUrl": "/favicon.png",
     "appName": "Pangea testnet",
-    "ecosystemName": "Pangea testnet",
+    "ecosystemName": "Pangea",
```
Comment (theblockstalk) [inline src/common/config/config.testnet.json:5]: ecosystem name: Pangea Testnet

File: src/common/config/config.testnet.json [hunk 7]
```diff
@@ -1,17 +1,13 @@
...
-    "theme": {
-        "primaryColor": "#6200ee",
-        "secondaryColor": "#03dac4",
-        "tertiaryColor": "white",
-        "linkColor": "#0057B2"
-    },
-    "appLogoUrl": "/tonomy-logo48.png",
+    "themeFile": "theme.pangea.css",
+    "appLogoUrl": "/favicon.png",
     "appName": "Pangea testnet",
```
Comment (theblockstalk) [inline src/common/config/config.testnet.json:4]: appName: Pangea Testnet

or better would be "Pangea Passport - Testnet". But if you need to change all of the infrastructure again For this then don't worry

File: index.html [hunk 8]
```diff
@@ -3,9 +3,7 @@
 
 <head>
   <meta charset="UTF-8" />
-  <link rel="icon" type="image/svg+xml" href="/tonomy-logo48.png" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Tonomy</title>
```
Comment (theblockstalk) [inline index.html:8]: shouldn't this also be set in settings.ts like the icon?

File: src/demo/pages/UserHome.tsx [hunk 9]
```diff
@@ -47,19 +48,22 @@ const USerHome: React.FC = () => {
...
-        administrators. As a user, Tonomy ID enables you access to a variety of
+        Our demo site showcases the benefits of {settings.config.ecosystemName}{" "}
+        for both users and administrators. As a user,{" "}
+        {settings.config.ecosystemName} enables you access to a variety of
         features. Some you can test on our demo website:
       </p>
       <CodeSnippetPreview
         snippetCode={snippetCode}
-        documentationLink="https://docs.tonomy.foundation/start/single-sign-on/#3-callback-page"
+        documentationLink={`${settings.config.documentationLink}/start/single-sign-on#id-3.-callback-page`}
```
Comment (theblockstalk) [inline src/demo/pages/UserHome.tsx:66]: will this also work with Gitbook?

File: src/demo/pages/BlockchainTransaction/SignTransactionSendPayment.tsx [hunk 10]
```diff
@@ -19,6 +19,8 @@ import {
 import useErrorStore from "../../../common/stores/errorStore";
 import settings from "../../../common/settings";
 
+const env = import.meta.env.VITE_APP_NODE_ENV || "development";
```
Comment (theblockstalk) [inline src/demo/pages/BlockchainTransaction/SignTransactionSendPayment.tsx:22]: shouldnt you get this from settings.ts ?

File: src/demo/pages/Home.tsx [hunk 11]
```diff
@@ -74,16 +72,19 @@ export default function Home() {
...
                 <img
-                  src={mobileLogo}
+                  src={settings.config.images.logo48}
+                  className="display-none"
+                  alt="Tonomy-logo"
+                />
+                <img
+                  src={settings.config.images.mobileLogo}
                   className="mobile-logo"
                   alt="Tonomy-logo"
```
Comment (theblockstalk) [inline src/demo/pages/Home.tsx:83]: should probably fix this alt text as well.

File: index.html (hunk 8 above)
Comment (sadiabbasi) [reply index.html:8]: it's already there document.title = settings.config.appName;

File: src/demo/pages/UserHome.tsx (hunk 9 above)
Comment (sadiabbasi) [reply src/demo/pages/UserHome.tsx:66]: i think it will work

File: src/accounts/pages/DownloadApp.tsx [hunk 12]
```diff
@@ -9,18 +9,21 @@ import playStoreBadge from "../assets/google-play-badge.png";
...
+        {settings.config.appName} is a self-sovereign identity digital
+        wallet that protects your security and privacy. &nbsp;
         <TLink href={settings.config.links.readMoreDownload} target="_blank">
           Learn more
         </TLink>
       </TP>
 
       <TP className="margin-bottom paragraph">
-        It is open-source and maintained by the Tonomy Foundation.{" "}
+        It is open-source and maintained by the {settings.config.appName}.{" "}
```
Comment (theblockstalk) [inline src/accounts/pages/DownloadApp.tsx:26]: this is not the appName. Please read the text.

File: src/accounts/pages/Login.tsx [hunk 13]
```diff
@@ -479,7 +483,7 @@ export default function Login() {
                 <TH3>{app.appName}</TH3>
                 <TH4>wants you to log in to their application</TH4>
                 <TP style={{ margin: "10px" }}>
-                  Please proceed to login to Tonomy ID app on your phone.
+                  Please proceed to login to Pangea app on your phone.
```
Comment (theblockstalk) [inline src/accounts/pages/Login.tsx:486]: appName

File: src/common/config/config.json [hunk 14]
```diff
@@ -1,17 +1,13 @@
 {
-    "theme": {
-        "primaryColor": "#67D7ED",
-        "secondaryColor": "#03dac4",
-        "tertiaryColor": "white",
-        "linkColor": "#0057B2"
-    },
-    "appLogoUrl": "/tonomy-logo48.png",
+    "themeFile": "theme.pangea.css",
+    "appLogoUrl": "/favicon.png",
```
Comment (theblockstalk) [inline src/common/config/config.json:3]: app logo should also be white labelable, depending on the environment.

File: src/common/config/config.staging.json [hunk 15]
```diff
@@ -1,17 +1,13 @@
...
-        "secondaryColor": "#03dac4",
-        "tertiaryColor": "white",
-        "linkColor": "#0057B2"
-    },
+    "themeFile": "theme.tonomy.css",
     "appLogoUrl": "/tonomy-logo48.png",
-    "appName": "Tonomy ID Staging",
-    "ecosystemName": "Tonomy Staging",
+    "appName": "Tonomy ID",
+    "ecosystemName": " Tonomy Staging",
```
Comment (theblockstalk) [inline src/common/config/config.staging.json:5]: "Tonomy Staging"

File: src/common/config/config.json (hunk 14 above)
Comment (sadiabbasi) [reply src/common/config/config.json:3]: it is fetching from public directory

**Pull request merged**